requirements.txt

```diff
@@ -1,50 +1,73 @@
 asgiref==3.7.2
+attrs==23.2.0
+autobahn==23.6.2
+Automat==22.10.0
+awsebcli==3.20.10
+bcrypt==4.1.2
 black==24.2.0
 blessed==1.20.0
 boto==2.49.0
 boto3==1.34.73
-botocore==1.34.73
+botocore
 cement==2.8.2
 certifi==2024.2.2
+cffi==1.16.0
+channels==4.1.0
+chardet==5.2.0
 charset-normalizer==3.3.2
 click==8.1.7
 colorama==0.4.3
-cryptography==42.0.5
+constantly==23.10.4
 coverage==6.5.0
 coveralls==3.3.1
+cryptography==42.0.5
+daphne==4.1.1
 distlib==0.3.8
 Django==4.2
 django-storages==1.14.2
 docopt==0.6.2
 filelock==3.13.1
 flake8==7.0.0
+hyperlink==21.0.0
 idna==3.6
+incremental==22.10.0
 jmespath==1.0.1
 mccabe==0.7.0
 mypy-extensions==1.0.0
+mysql-connector==2.2.9
+mysql-connector-python==8.3.0
 numpy==1.26.2
 packaging==23.2
 pandas==2.1.3
 pathspec==0.10.1
+pillow==10.2.0
 pipenv==2023.11.14
 platformdirs==3.11.0
+pyasn1==0.6.0
+pyasn1_modules==0.4.0
 pycodestyle==2.11.1
+pycparser==2.21
 pyflakes==3.2.0
 PyJWT==1.7.1
 PyMySQL==1.1.0
+pyOpenSSL==24.1.0
 python-dateutil==2.8.2
 pytz==2024.1
 PyYAML==6.0.1
+reportlab==4.1.0
 requests==2.31.0
 s3transfer==0.10.1
 semantic-version==2.8.5
+service-identity==24.1.0
 six==1.16.0
 sqlparse==0.4.4
 termcolor==1.1.0
 tomli==2.0.1
+Twisted==24.3.0
+txaio==23.1.1
 typing_extensions==4.9.0
 tzdata==2024.1
 urllib3==1.26.18
 virtualenv==20.24.6
 wcwidth==0.1.9
-reportlab~=4.1.0
+zope.interface==6.2
```

Comment:
Please add the reportlab package. Without that, the report generation might not work. @cindyke124 could you please confirm?